cpp/src/slice2py/PythonUtil.cpp

```diff
@@ -1503,411 +1896,270 @@ Slice::Python::CodeVisitor::visitEnum(const EnumPtr& p)
     {
-        _out << getTypeReference(constant);
+        out << getImportAlias(constant);
     }
-    else
+    else if (auto builtin = dynamic_pointer_cast<Slice::Builtin>(type))
     {
-        Slice::BuiltinPtr b = dynamic_pointer_cast<Slice::Builtin>(type);
-        Slice::EnumPtr en = dynamic_pointer_cast<Slice::Enum>(type);
-        if (b)
+        switch (builtin->kind())
         {
-            switch (b->kind())
+            case Slice::Builtin::KindBool:
             {
-                case Slice::Builtin::KindBool:
-                {
-                    _out << (value == "true" ? "True" : "False");
-                    break;
-                }
-                case Slice::Builtin::KindByte:
-                case Slice::Builtin::KindShort:
-                case Slice::Builtin::KindInt:
-                case Slice::Builtin::KindFloat:
-                case Slice::Builtin::KindDouble:
-                case Slice::Builtin::KindLong:
-                {
-                    _out << value;
-                    break;
-                }
-                case Slice::Builtin::KindString:
-                {
-                    const string controlChars = "\a\b\f\n\r\t\v";
-                    const unsigned char cutOff = 0;
+                out << (value == "true" ? "True" : "False");
+                break;
+            }
+            case Slice::Builtin::KindByte:
+            case Slice::Builtin::KindShort:
+            case Slice::Builtin::KindInt:
+            case Slice::Builtin::KindFloat:
+            case Slice::Builtin::KindDouble:
+            case Slice::Builtin::KindLong:
+            {
+                out << value;
+                break;
+            }
+            case Slice::Builtin::KindString:
+            {
+                const string controlChars = "\a\b\f\n\r\t\v";
+                const unsigned char cutOff = 0;
 
-                    _out << "\"" << toStringLiteral(value, controlChars, "", UCN, cutOff) << "\"";
-                    break;
-                }
-                case Slice::Builtin::KindValue:
-                case Slice::Builtin::KindObject:
-                case Slice::Builtin::KindObjectProxy:
-                    assert(false);
+                out << "\"" << toStringLiteral(value, controlChars, "", UCN, cutOff) << "\"";
+                break;
             }
+            case Slice::Builtin::KindValue:
+            case Slice::Builtin::KindObject:
+            case Slice::Builtin::KindObjectProxy:
+                assert(false);
         }
-        else if (en)
-        {
-            EnumeratorPtr lte = dynamic_pointer_cast<Enumerator>(valueType);
-            assert(lte);
-            _out << getTypeReference(lte);
-        }
-        else
-        {
-            assert(false); // Unknown const type.
-        }
+    }
+    else if (auto enumeration = dynamic_pointer_cast<Slice::Enum>(type))
+    {
+        EnumeratorPtr enumerator = dynamic_pointer_cast<Enumerator>(valueType);
```

Comment:
Just as a general comment, this PR sometimes qualify things with `Slice::`, other times it doesn't.
Personally, I think it's fine to omit the `Slice::`, since there really shouldn't be any collisions and it keeps things looking tidier.